tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php

```diff
@@ -606,9 +606,7 @@ class Foo
 EOT;
 
         $this->fixer->configure($configuration);
-        $this->doTest($expected, $input);
 
-        $this->fixer->configure(array('elements' => $configuration));
```

Comment:
I think it is better not to test two thinks in one test, therefor I think it is better to move this into a new test method.